packages/tradershub/src/components/Base/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useCopyToClipboard, useHover } from 'usehooks-ts';
+import { Button, useBreakpoint } from '@deriv/quill-design';
+import ClipboardIcon from '../../../public/images/clipboard.svg';
+import CheckmarkCircle from '../../../public/images/ic-checkmark-circle.svg';
+import { Tooltip } from '../Tooltip';
+
+type TClipboardProps = {
+    textCopy: string;
+};
+
+/**
+ * Props for the Clipboard component.
+ * @property {string} textCopy - The text to be copied to the clipboard.
+ *
+ * @example
+ * ```jsx
+ * <Clipboard textCopy="Text to be copied" />
+ * ```
+ */
+const Clipboard = ({ textCopy }: TClipboardProps) => {
+    const [, copy] = useCopyToClipboard();
```

Comment:
```suggestion
    const [value, copy] = useCopyToClipboard();
```

Cant you use value to know if the user has copied something? Something like
```
value === textCopy
```
would tell if the user copied something or not?

Reply:
yes it become a green check icon: ![image](https://github.com/binary-com/deriv-app/assets/108507236/aebba794-2bd0-4e44-bc26-c885a225f399) [file in PR](https://github.com/binary-com/deriv-app/pull/12427/files#diff-aa70f8504e8e9fd6bcd511edff5e89a8fa073ce1f969caa23bf1b07559767fa5)

Reply:
yes it would tell (by the 'copied!' message on line 47)
<img width="76" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/b65f19a0-1275-48d4-b31e-65f0020eda82">

we can declare the `value` but it would be an unused variable unless we are using it somewhere in the component like  ```<Tooltip message={isCopied ? `Copied! ${value}` : 'Copy'}>``` which we are not

Reply:
I see I may have misunderstood your suggestion

using `value === textCopy` would mean `isCopied` will remain true since the value won't be unset, hence, the green check icon are not reverting back to clipboard icon